cypress/e2e/onboarding.spec.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
+
+describe('Onboarding Page', () => {
+  afterEach(() => {
+    cy.truncateDatabase();
+  });
+
+  it('displays the correct heading', () => {
+    cy.login();
+    cy.visit('http://localhost:3000/onboarding');
+    cy.contains('h1', 'Onboarding');
+  });
+
+  it('fills out the form successfully', () => {
+    cy.get('form').within(() => {
+      cy.get('textarea[name="bio"]').type('Lorem ipsum dolor sit amet.');
```

Comment:
Too much was removed here, thus causing the tests to fail.

You had:
```js
    cy.visit('http://localhost:3000');
    cy.login();
    cy.visit('http://localhost:3000/onboarding');
    [...]
```
Only the first `cy.visit()` should have been removed, so:
```js
    cy.login();
    cy.visit('http://localhost:3000/onboarding');
    [...]
```